querycheck/known_value.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package querycheck
+
+import (
+	"github.com/hashicorp/terraform-plugin-testing/knownvalue"
+	"github.com/hashicorp/terraform-plugin-testing/tfjsonpath"
+)
+
+type KnownValueCheck struct {
```

Comment:
Since this is an external package, a little documentation blurb about what this struct is / how it relates to resource objects found via query filters, might be useful 👍🏻